README.md

```diff
@@ -1,73 +1,49 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# To Do List App
 
+> A simple To Do List App for learning purpose
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- Css
+- HTML
+- Webpack
+- Javascript
+## Live Demo
 
+[Live Demo Link](https://yayner2002.github.io/to-do-list-app/)
 
+## Screenshot
+![home page](app_screenshot.png?raw=true "home screenshot")
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
```

Comment:
Please you will need to add some lines of code so that other partners can clone your repository locally